### DISCUSSION TITLE
Add possibility of private, personal collections

### DISCUSSION THREAD

**Discussion by magnuslo** [outline/outline#7665] [Ideas]
My team and I love using outline to co-operate but some times we find the need to have private docs only visible to ourselves. We've solved this by making a new collection with the privacy turned to `No access`. It would be nice to have this as a fully fledged feature, and this is perhaps not so difficult to do? 

Notion already has this when signed in to a team space.

**Comment by tommoor** [comment]
Hi Magnus, as you've found you can already achieve this by just creating a private collection – is there anything from the Notion implementation in particular you're missing? I think we could consider moving these to their own section in the sidebar

**Reply by magnuslo** [reply]
It would be neater and more organized to have these under its own title, "Private collections" for instance.

**Reply by tommoor** [reply]
Agreed

**Reply by tovoro** [reply]
+1, would be easier to have a standard collection "personal" or similar created for every user automatically. 
The reason is  I would like to disable collection creation for the users but enable them to have a personal one anyway, at the moment, both together is not possible, if im not mistaken?

**Reply by CanalWood** [reply]
+1, I would like to have a personal collection for private meeting note. I'can't use a new collection with the privacy turned to "No access" because at leat one user must have manage permission and it's herited. It's not secure for a personnal access.

<img width="1920" height="945" alt="image" src="https://github.com/user-attachments/assets/d1055af1-9d8d-48fc-bc88-c8474a930ae0" />

Any roadmap news about this topic?
